src/pages/home/ReportScreen.js

```diff
@@ -140,7 +152,7 @@ class ReportScreen extends React.Component {
     prepareTransition() {
         this.setState({isLoading: true});
         clearTimeout(this.loadingTimerId);
-        this.loadingTimerId = setTimeout(() => this.setState({isLoading: false}), 0);
+        this.loadingTimerId = setTimeout(() => this.setState({isLoading: false}), 100);
```

Comment:
I don't think causing an intentional delay creates a good UX. You can confirm this from the design team.

Reply:
**question:** Why were we previously setting a delay here?

**question:** Why are we increasing this delay now?

Reply:
> question: Why were we previously setting a delay here?

Not sure about this.

> question: Why are we increasing this delay now?

Switching between the chats doesn't show the loader and its an instant switch. To make the transition smoother I've added the delay to test. I am guessing its the same reason we added the delay earlier. I've rolled back to original. Will need 👀  on this one.

Reply:
Thanks, only change things that you can understand and defend please otherwise ask for help.